### PR TITLE
feat(Form): add `inverted` prop

### LINKED ISSUE #1212: feat(Form): add `inverted` boolean property
As seen on the official Semantic-UI page here: 
http://semantic-ui.com/collections/form.html

a `<form>` is able to be inverted, for use on dark backgrounds.

On the semantic ui react documentation page for forms here:
http://react.semantic-ui.com/collections/form

there is no mention of this usecase. `inverted` is also missing under `Props` for `Form`. Adding `inverted` to a `Form` element does not affect it in any way either.

### PR DESCRIPTION
**Pull request by vageeshb**
* Added inverted prop for form
* Added inverted form doc example
* Fixes #1212

### CODE REVIEW

File: docs/app/Examples/collections/Form/Variations/FormExampleInverted.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Button, Form, Segment } from 'semantic-ui-react'
+
+
+const FormExampleInverted = () => (
+  <Segment inverted={true}>
+    <Form inverted={true}>
```
Comment (levithomason) [inline docs/app/Examples/collections/Form/Variations/FormExampleInverted.js]: Boolean props are true by default, let's go with `inverted` over `inverted={true}` 👍

File: docs/app/Examples/collections/Form/Variations/index.js [hunk 2]
```diff
@@ -9,6 +9,11 @@ const FormFormVariationsExamples = () => (
       description='A form can vary in size'
       examplePath='collections/Form/Variations/FormExampleSize'
     />
+    <ComponentExample
+      title='Inverted'
+      description='A form on a dark background may have to invert its color scheme'
```
Comment (levithomason) [inline docs/app/Examples/collections/Form/Variations/index.js]: Period after description please, we're currently trying to standardize this.

File: src/collections/Form/Form.js [hunk 3]
```diff
@@ -258,11 +261,13 @@ export default class Form extends Component {
       success,
       warning,
       widths,
+      inverted,
```
Comment (levithomason) [inline src/collections/Form/Form.js]: Please alphabetically sort the destructured props.

**Comment by levithomason** [comment]
Thanks much for the PR!  This looks great, made a couple minor comments for you.

**Comment by layershifter** [comment]
Please, don't forget about typings.

**Comment by vageeshb** [comment]
@levithomason Updated with feedback
@layershifter Haven't worked with TS, added typings. Please verify.

File: docs/app/Examples/collections/Form/Variations/FormExampleInverted.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Button, Form, Segment } from 'semantic-ui-react'
+
+
+const FormExampleInverted = () => (
+  <Segment inverted>
+    <Form inverted>
+      <Form.Group widths='equal'>
+        <Form.Field label='First name' control='input' placeholder='First name' />
+        <Form.Field label='Last name' control='input' placeholder='Last name' />
```
Comment (levithomason) [inline docs/app/Examples/collections/Form/Variations/FormExampleInverted.js]: Apologies that I missed this on the first pass.  Let's use the shorthand `<Form.Input />` for brevity.  I like to keep the doc examples as slim as possible:

```diff
-<Form.Field label='First name' control='input' placeholder='First name' />
+<Form.Input label='First name' placeholder='First name' />
```

File: test/specs/collections/Form/Form-test.js [hunk 5]
```diff
@@ -37,6 +37,7 @@ describe('Form', () => {
   common.propKeyOnlyToClassName(Form, 'reply')
   common.propKeyOnlyToClassName(Form, 'success')
   common.propKeyOnlyToClassName(Form, 'warning')
+  common.propKeyOnlyToClassName(Form, 'inverted')
```
Comment (levithomason) [inline test/specs/collections/Form/Form-test.js]: Nit: alphabetical by prop name please.  I realize many are out of order, however, there is a current effort to update these.

File: src/collections/Form/Form.js [hunk 6]
```diff
@@ -263,6 +267,7 @@ export default class Form extends Component {
     const classes = cx(
       'ui',
       size,
+      useKeyOnly(inverted, 'inverted'),
```
Comment (levithomason) [inline src/collections/Form/Form.js]: Nit: alphabetical by prop name please

**Review by levithomason — CHANGES_REQUESTED** [review]

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
Thanks for the extra fixes!

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.64.4`.

**Pull request merged**